engine/src/main/scala/cromwell/webservice/PartialWorkflowSources.scala

```diff
@@ -58,6 +60,9 @@ object PartialWorkflowSources {
 
     val partialSources: ErrorOr[PartialWorkflowSources] = {
       def getStringValue(key: String) = formData.get(key).map(_.utf8String)
+      def getBooleanValue(key: String) = getStringValue(key) map { b =>
+        Try(b.toBoolean).toErrorOr
+      }
```

Comment:
I think this could be `getStringValue(key).map(Try(_.toBoolean).toErrorOr)`

Reply:
I tried the way you suggested. It could not resolve 'toBoolean' function. So I have changed it to getStringValue(key).map(b => Try(b.toBoolean).toErrorOr)